tests/unit/modules/network/sonic/test_sonic_lldp_interfaces.py

```diff
@@ -47,6 +47,25 @@ def tearDown(self):
         self.mock_config_edit_config.stop()
         self.mock_get_interface_naming_mode.stop()
 
+    def validate_config_requests(self):
+        lldp_path = 'data/openconfig-lldp:lldp/interfaces/interface=Ethernet{}/config/openconfig-lldp-ext:suppress-tlv-advertisement'
+        interface_number = None
+        for request in self.config_requests_valid:
+            if request['path'] == lldp_path:
```

Comment:
```suggestion
            if "openconfig-lldp-ext:suppress-tlv-advertisement" in request['path']:
``` 
I feel you should do a substring check here instead of a full exact match check. That should address Kerry's concern about the varieties of tlv suppression.